config/locales/en.yml

```diff
@@ -152,7 +152,7 @@ en:
     assign_self: Assign myself an article
     confirm_add_available: Are you sure you want to add %{title}?
     confirm_addition: Are you sure you want to assign %{title} to %{username}?
-    confirm_deletion: Are you sure you want to delete this assignment?
+    confirm_deletion: Are you sure you want to delete this article?
```

Comment:
I think this is a little unclear: it risks confusion about deleting the Wikipedia article itself. I suggest 'article assignment'.